hunts/src/index.js

```diff
@@ -1 +1,2 @@
+import "regenerator-runtime/runtime.js";
```

Comment:
Is this needed? Does React/Redux require generator functions, or are you using any?